c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -83,6 +89,19 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: this.compile_(template)(this.scope_)});
 };
 
+/**
+ * @param {Object} doc
+ * @export
```

Comment:
This function looks private (only used in the controller and with a trailing "_" in the name): ``@export`` => ``@private``